tests/python/gaia-ui-tests/gaiatest/apps/system/regions/fxaccounts.py

```diff
@@ -68,6 +70,7 @@ def wait_for_successful_login(self):
         return done
 
     def tap_done(self):
+        self._switch_to_fxa_iframe()
         done = self.wait_for_successful_login()
         done.tap()
         #Switch back to the settings app
```

Comment:
It's fine, but can you mention it when you ask review. I was confused by this change.